grantnav/frontend/tests_functional.py

```diff
@@ -28,9 +28,9 @@ def browser(request):
         chrome_options.add_argument("--headless")
         # no-sandbox prevents an error when running as the root user
         chrome_options.add_argument("--no-sandbox")
-        # uncomment this if "DevToolsActivePort" error
-        # chrome_options.add_argument("--remote-debugging-port=9222")
-
+        # uncomment this if "DevToolsActivePort" error / ubuntu snap workaround
+        chrome_options.add_argument("--remote-debugging-port=9222")
```

Comment:
Re-comment this workaround?

I copy-n-paste a whole bunch of code to make it use Firefox every time I test, there must be a better way...

Reply:
Argh yeah, I'm tempted to leave that in since it works in CI and I don't think it does any harm? Maybe a better way is to have our own .selenium.conf or something, it's a bit of a nightmare though as they're liable to change.